RELEASING/release-notes-1-5/README.md

```diff
@@ -0,0 +1,136 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes for Superset 1.5
+
+Superset 1.5 focuses on polishing the dashboard native filters experience, while
+improving performance and stability. Superset 1.5 is likely the last minor release of
+version 1 of Superset, and will be succeeded by Superset 2.0. The 1.5 branch
+introduces the notion of a Long Term Support (LTS) version of Superset, and will
+receive security and other critical fixes even after Superset 2.x is released.
+Therefore, users will have the choice of staying on the 1.5 branch or upgrading to 2.x
+when available.
+
+- [**User Experience**](#user-facing-features)
+- [**Feature flags**](#feature-flags)
+- [**Database Experience**](#database-experience)
+- [**Breaking Changes and Full Changelog**](#breaking-changes-and-full-changelog)
+
+## User Facing Features
+
+- Complex dashboards with lots of native filters and charts will render considerably
+  faster. See the videos that shows the rendering time of a complex dashboard go from
+  11 to 3 seconds ([#19064](https://github.com/apache/superset/pull/19064)). The Native
+  Filter Bar has been redesigned, along with moving the "Apply" and "Clear all" buttons
+  to the bottom:
+
+![Filter bar](media/filter_bar.png)
+
+- Native filters can now define multiple parent filters, making it possible to do arbitrarily
```

Comment:
We actually wanted to avoid "parent" and "cascading" - we even renamed all variables in order not to use those names. Can we use some variants of "dependent on" instead?

Reply:
Sure; would this formulation work? 
> Native filters can now be made dependent on multiple filters. This makes it possible to restrict the available values in a filter based on the selection of other filters.

Reply:
Yes, that's perfect 🙂